app/src/components/notifications/LNotification.vue

```diff
@@ -51,4 +96,27 @@ const show = ref(true);
             </div>
         </div>
     </div>
+
+    <!-- Banner -->
+    <div
+        v-if="show && notification.type == 'banner'"
+        class="inset-x-0 top-0 z-50 text-zinc-900"
+        :class="colour"
+    >
+        <div class="flex items-center justify-between px-6 py-1">
+            <div class="flex items-center gap-2">
+                <component :is="icon" class="h-5 min-h-5 w-5 min-w-5" />
+                <div class="flex flex-col md:inline-block md:align-middle">
+                    <span class="text-md md:text-sm">{{ notification.title }}</span>
+                    <span v-if="notification.description" class="text-xs md:ml-3">{{
+                        notification.description
+                    }}</span>
+                </div>
+            </div>
+            <XMarkIcon
+                @click="notification.id ? removeNotification(notification.id) : (show = false)"
+                class="h-6 min-h-6 w-6 min-w-6 cursor-pointer underline md:h-5 md:min-h-5 md:w-5 md:min-w-5"
+            />
+        </div>
+    </div>
```

Comment:
Again here I think this belongs in its own component. The only code that's shared here is the `icon`, and that could be extracted to a different place - perhaps the store?

I suggest to split this in `LToast` and `LBanner` components, or just `NotificationToast` and `NotificationBanner`.